app/controllers/application_controller.rb

```diff
@@ -2,4 +2,8 @@ class ApplicationController < ActionController::Base
   # def command_injection
   #   system(params[:command])
   # end
+
+  def hoge(str1)
+    str1.start_with?('a') || str1.start_with?('b')
```

Comment:
<article><p><b>[Rubocop]</b> <span>Use `str1.start_with?('a', 'b')` instead of `str1.start_with?('a') || str1.start_with?('b')`.</span></p><details><p></p><p>7 line , 5 column in <code>app/controllers/application_controller.rb</code></p><dl><dt>Rule</dt><dd>Performance/DoubleStartEndWith</dd><dt>Severity</dt><dd>convention</dd></dl></details></article>